TombEngine/Sound/sound.h

```diff
@@ -38,6 +38,18 @@ enum class SoundTrackType
 	Count
 };
 
+enum SampleRate
```

Comment:
This enum does not make sense. TEN internally resamples a sound according to a pitch value in sound info. If you allow 2 separate sample rates, a major sound engine refactor must be performed so that engine knows original sample rate for every loaded sample. We won't dig into that now. Remove this.